### DISCUSSION TITLE
My site, rockett.pw

### DISCUSSION THREAD

**Discussion by mikerockett** [ryangjchandler/orbit#71] [Show and tell]
I've been through a bunch of platforms and content systems for my site, and none of them were as developer friendly as this. And I think it's because it's not a CMS in the first place – it's just an Eloquent tool that makes life so much easier. But then, couple it with Filament, and it's a dream! Built my very basic [v4 site](https://rockett.pw/) in under 3 hours!

Just wanted to say thanks for this – it works super well. I noticed caching issues with the sqlite DB when adding files manually, and not through the event system. Would be nice if it could detect new data efficiently. But, the second you add a frontend that does the job, the issue isn't present anymore, and everything just works.

**Comment by ryangjchandler** [comment]
Glad to hear it @mikerockett! I've got some plans to improve the caching for v1.0, meaning the entire SQLite database won't need to be refreshed each time a piece of data changes.

I'll add your site to my uses list and start adding them to the README too. Again, thanks for using the library!
